plugins/tours/src/admin/wp-list-table.php

```diff
@@ -15,11 +15,12 @@
 /**
  * Add custom columns to the Tours admin page.
  *
- * @return array Array of columns.
  * @since 1.0.0
+ * @param array[] $post_columns An associative array of column headings.
  *
+ * @return array An array of custom column headings.
  */
-function set_custom_columns() {
+function set_custom_columns( $post_columns ) {
 	return array(
```

Comment:
Change `array()` to short form `[]` for consistency.

Reply:
@rgadon107 Fixed.